### DISCUSSION TITLE
How I can build version for future versions of blender? WINDOWS

### DISCUSSION THREAD

**Discussion by Whhhatttt** [JacquesLucke/animation_nodes#1930] [Q&A]
Can someone attach link or explain how to build animation_nodes for newer versions of blender? I Tried to find, but didn't find..

FOR WINDOWS! Not for mac. Pls. What should you pay attention to when building? And where and how can you generally update the code structure so that in general everything works. I would be extremely grateful if you tell me how to do this! 

P.S: 
![image](https://github.com/JacquesLucke/animation_nodes/assets/40802556/dbdba4b8-4cc4-4d8b-bda9-7623e8a04ea2)
How I can change this color? (red), Cant find same RGB or HEX in python scripts..

**Comment by OmarEmaraDev** [comment]
See https://github.com/JacquesLucke/animation_nodes_manual/blob/v2.0/docs/dev_guide/setup/windows.rst.

**Comment by bogdan-peregubko** [comment]
Is process the same for building for macOS?

**Reply by OmarEmaraDev** [reply]
Well, you can install Cython from homebrew or pip instead of Conda, but the command is the same.
